### ISSUE TITLE
Windows doesn't recognise SNES mini, but sees it.

### ISSUE THREAD

**Issue by daunidebil** [ClusterM/hakchi2#1575]
I hacked my mini back in 2022, on my laptop, but after some time i got a PC and transfered all my files except hakchi2 files, and sold my laptop. There was no way to restore the old data, so i tried to flash the original kernel, but when i tried to do that, it kept asking me to download drivers, i tried to do that, but it kept failing after some time. Windows just says that its "unknown device", and after the driver "install", it said also "driver not available". Is there anything to do about this? SNES still works, but it's just like unmoddable now.

**Comment by ClusterM** [comment]
Hi.
Can you see this device in the device manager? Also, try another USB cable/port.

**Comment by daunidebil** [comment]
@ClusterM yes i can see it in the device manager, tried all of my usb cables and ports.
